scripts/google-java-format.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Array of glob patterns
+glob_patterns=(
+    "samples/react-native/android/app/src/**/*.java"
+    "packages/core/android/**/*.java"
+    "performance-tests/TestAppPlain/android/app/src/**/*.java"
+    "performance-tests/TestAppSentry/android/app/src/**/*.java"
+)
+
+# Check if an argument is provided
+if [ $# -eq 0 ]; then
+    echo "Usage: $0 <fix|lint>"
+    exit 1
+fi
+
+# Set the mode based on the first argument
+mode=$1
+
+# Base command
+base_cmd="npx google-java-format"
+
+# Add --replace flag if mode is 'fix'
+if [ "$mode" = "fix" ]; then
+    base_cmd+=" --replace"
+elif [ "$mode" = "lint" ]; then
+    base_cmd+=" --set-exit-if-changed"
```

Comment:
Thank you for this PR @krystofwoldrich 🙇 

I'm seeing the following error on CI but linting finishes successfully on my local machine.

```
Executing: npx google-java-format --set-exit-if-changed --glob='packages/core/android/**/*.java'
Error: google-java-format exited with exit code 1.
    at errorFromExitCode (/home/runner/work/sentry-react-native/sentry-react-native/node_modules/google-java-format/index.js:19:10)
    at ChildProcess.<anonymous> (/home/runner/work/sentry-react-native/sentry-react-native/node_modules/google-java-format/index.js:111:40)
    at ChildProcess.emit (node:events:517:28)
    at maybeClose (node:internal/child_process:1098:16)
    at ChildProcess._handle.onexit (node:internal/child_process:303:5)
Command failed: npx google-java-format --set-exit-if-changed --glob='packages/core/android/**/*.java'
```

I suggest adding the `dry-run` option to get more data on any environment differences that may cause the failure (e.g. eol etc).
```suggestion
    base_cmd+=" --set-exit-if-changed --dry-run"
```

Reply:
Thank you.

I added diff output to see what code was changed.

```
Executing: npx google-java-format --set-exit-if-changed --replace --glob='samples/react-native/android/app/src/**/*.java'
Executing: npx google-java-format --set-exit-if-changed --replace --glob='packages/core/android/**/*.java'
Error: google-java-format exited with exit code 1.
    at errorFromExitCode (/home/runner/work/sentry-react-native/sentry-react-native/node_modules/google-java-format/index.js:19:10)
    at ChildProcess.<anonymous> (/home/runner/work/sentry-react-native/sentry-react-native/node_modules/google-java-format/index.js:111:40)
    at ChildProcess.emit (node:events:517:28)
    at maybeClose (node:internal/child_process:1098:16)
    at ChildProcess._handle.onexit (node:internal/child_process:303:5)
Command failed: npx google-java-format --set-exit-if-changed --replace --glob='packages/core/android/**/*.java'
Executing: npx google-java-format --set-exit-if-changed --replace --glob='performance-tests/TestAppPlain/android/app/src/**/*.java'
Executing: npx google-java-format --set-exit-if-changed --replace --glob='performance-tests/TestAppSentry/android/app/src/**/*.java'
Printing git patch for currently changed tracked files:
diff --git a/packages/core/android/src/main/java/io/sentry/react/RNSentryModuleImpl.java b/packages/core/android/src/main/java/io/sentry/react/RNSentryModuleImpl.java
index a4dab7a..85e5faa 100[64](https://github.com/getsentry/sentry-react-native/actions/runs/11346713597/job/31556423248?pr=4175#step:6:68)4
--- a/packages/core/android/src/main/java/io/sentry/react/RNSentryModuleImpl.java
+++ b/packages/core/android/src/main/java/io/sentry/react/RNSentryModuleImpl.java
@@ -922,8 +922,8 @@ public class RNSentryModuleImpl {
     SdkVersion sdk = event.getSdk();
     if (sdk != null) {
       switch (sdk.getName()) {
-        // If the event is from capacitor js, it gets set there and we do not handle it
-        // here.
+          // If the event is from capacitor js, it gets set there and we do not handle it
+          // here.
         case NATIVE_SDK_NAME:
           setEventEnvironmentTag(event, "native");
           break;
One or more commands failed.
ERROR: "lint:android" exited with 1.
```